### PR TITLE
CHE-747: Split GWT compilation and wsmaster API war

### PR DESCRIPTION
**Pull request by vparfonov**
**_1 Upvote**_

### CODE REVIEW

**Comment by vparfonov** [comment]
@benoitf  @skabashnyuk @gazarenkov @TylerJewell

**Comment by skabashnyuk** [comment]
ок

**Comment by gazarenkov** [comment]
ok

**Comment by benoitf** [comment]
so cool ! +1

File: assembly/assembly-wsmaster-war/pom.xml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-assembly-parent</artifactId>
+        <groupId>org.eclipse.che</groupId>
+        <version>4.3.0-RC1-SNAPSHOT</version>
+    </parent>
+    <artifactId>assembly-wsmaster-war</artifactId>
+    <packaging>war</packaging>
+    <name>Che IDE :: Compiling WS Master WAR</name>
```
Comment (sleshchenko) [inline assembly/assembly-wsmaster-war/pom.xml:23]: Is it compiling or packaging like https://github.com/eclipse/che/pull/1216/files#diff-d4c886c1cbe35ec2bcc892d158db1014R23 ?

**Comment by garagatyi** [comment]
+1 it will improve speed of server side development!

File: assembly/assembly-wsmaster-war/pom.xml (hunk 1 above)
Comment (vparfonov) [inline assembly/assembly-wsmaster-war/pom.xml:23]: ok will fix

**Comment by TylerJewell** [comment]
Verify that /bin/install-plugin.sh still works for creating new assemblies.  Otherwise ok.  We will need to update docs extensively for 4.3 once this is merged.

**Pull request merged**